src/components/02-molecules/MeetingForm/index.js

```diff
@@ -23,10 +24,15 @@ const MeetingForm = ({
   validationErrors = {},
   visibleErrorMessages,
   isCreatingMeeting,
+  isEditingMeeting,
   handleDeleteClick,
   handleSaveClick,
  }) => {
+  const now = moment()
+  const meetingStart = moment(meeting.start)
+
   const isSubmitDisabled = invalid || hasValidationErrors(validationErrors)
+  const isStartDisabled = isEditingMeeting && meetingStart.isBefore(now)
```

Comment:
As discussed, I think it would make sense to have the time picker not be able to select a time that is earlier than now.